### PR TITLE
OPSEXP-1779 Add EC2 OpenSearch integration test

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1779

### CODE REVIEW

File: .github/workflows/enteprise.yml [hunk 1]
```diff
@@ -149,6 +149,9 @@ jobs:
           - name: multimachine
             vars: vars.yml
             desc: EC2 ACS 7.3 clustered (RHEL8)
+          - name: opensearch
+            vars: vars.yml
+            desc: EC2 ACS 7.3 clustered (RHEL8)
```
Comment (slohe1) [inline .github/workflows/enteprise.yml:154]: Can we give the ```desc``` more related to opensearch?

File: molecule/opensearch/hosts.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+all:
+  children:
+    external:
+      children:
+        external_elasticsearch:
+          hosts:
+            vpc-molecule-rhel8-opsexp-1779-0-rdnia4e7tivalj2n7z35in2e7e.eu-west-1.es.amazonaws.com:
+              {elasticsearch_port: 443, elasticsearch_protocol: https, elasticsearch_username: admin}
```
Comment (gionn) [inline molecule/opensearch/hosts.yml:8]: to revert?

File: molecule/opensearch/converge.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include alfresco-ansible-deployment"
+      include_role:
```
Comment (gionn) [inline molecule/opensearch/converge.yml:6]: 🤔 should'nt execute the entire playbook?

File: roles/repository/templates/alfresco-global.properties.j2 [hunk 4]
```diff
@@ -37,6 +37,8 @@ elasticsearch.indexName=alfresco
 elasticsearch.host={{ elasticsearch_host }}
 elasticsearch.port={{ ports_cfg.elasticsearch.http }}
 elasticsearch.secureComms={% if elasticsearch_protocol == 'https' %}https{% else %}none{% endif +%}
+elasticsearch.user={{ elasticsearch_username | default('') }}
+elasticsearch.password={{ elasticsearch_password | default('') }}
```
Comment (gionn) [inline roles/repository/templates/alfresco-global.properties.j2:41]: there are always good reasons to write integration tests

File: molecule/opensearch/hosts.yml (hunk 2 above)
Comment (alxgomz) [reply molecule/opensearch/hosts.yml:8]: indeed! it sneaked in my git index!

File: molecule/opensearch/converge.yml (hunk 3 above)
Comment (alxgomz) [reply molecule/opensearch/converge.yml]: leftovers from the molecule init

**Review by gionn — APPROVED** [review]

**Pull request merged**